src/containers/user/UserSettings.js

```diff
@@ -26,6 +26,8 @@ import Icon from '../../components/Icon'
 @prefetch(({ dispatch, params: { id }, query }) => {
   switch (query.expand) {
     case 'password':
+      dispatch(toggleUserSettingsSection('account', true))
+      return dispatch(toggleUserSettingsSection('password', true))
```

Comment:
reusing this action and store is a little hacky, but it also makes sense to me. Possibly the action needs renaming?